docs/devops.md

```diff
@@ -7,7 +7,7 @@
 
 Thanks for your interest in learning more about how we do DevOps for the platform at freeCodeCamp.org.
 
-This guide will help you understand our infrastructure stack and how we maintain our platforms. While, this is guide does not have exhaustive details for all operations, it could be used as a reference for your understanding of the systems.
+This guide will help you understand our infrastructure stack and how we maintain our platforms. While, this guide does not have exhaustive details for all operations, it could be used as a reference for your understanding of the systems.
```

Comment:
If we are cleaning up this sentence, let's remove the unnecessary comma.
```suggestion
This guide will help you understand our infrastructure stack and how we maintain our platforms. While this guide does not have exhaustive details for all operations, it could be used as a reference for your understanding of the systems.
```